### DISCUSSION TITLE
Dislotwin Setting of the self-diffusion coefficient parameter

### DISCUSSION THREAD

**Discussion by Asia-liu** [damask-multiphysics/DAMASK#19] [Q&A] [answered]
In the process of setting constitutive parameters in the reference article [1]: D represents the grain size, D_a represents (Canni) is a fitting parameter. What I want to ask is the Self-diffusion coefficient for fcc Fe of D_0. How is the D_0 parameter set? Thank you for your reply
[1]Wong S L, Madivala M, Prahl U, et al. A crystal plasticity model for twinning-and transformation-induced plasticity[J]. Acta Materialia, 2016, 118: 140-151.

**Comment by MarDiehl** [comment, marked as answer]
Hello,

the model was slightly changed, we now use a model from Argon & Moffat, Acta Metallurgica, Vol. 29, pg 293 to 299, 1981 to calculate the climb velocity. The implementation is here: https://github.com/damask-multiphysics/DAMASK/blob/28311158c069f23b363082b75844bf29c0889a52/src/phase_mechanical_plastic_dislotwin.f90#L702

**Reply by Asia-liu** [reply]
Ok thank you!
Your article is also very important to me
[Shah V, Sedighiani K, Van Dokkum J S, et al. Coupling crystal plasticity and cellular automaton models to study meta-dynamic recrystallization during hot rolling at high strain rates[J]. Materials Science and Engineering: A, 2022, 849: 143471.]
